clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/MasterElectionTest.java

```diff
@@ -226,24 +230,20 @@ public void testClusterStateVersionIncreasesAcrossMasterElections() throws Excep
         startingTest("MasterElectionTest::testClusterStateVersionIncreasesAcrossMasterElections");
         FleetControllerOptions options = defaultOptions("mycluster");
         options.masterZooKeeperCooldownPeriod = 1;
-        setUpFleetController(5, false, options);
+        setUpFleetController(3, false, options);
```

Comment:
I'm assuming this is to reduce the amount of load generated by the test?

Reply:
Yes, and we usually have 3 and not 5 controllers. But this is not important, I think I just tried it out